src/graph_compute/phase_scheduling.h

```diff
@@ -40,6 +40,8 @@ static inline bool interChunkDependency(vid_t v, vid_t w) {
   static const vid_t chunkMask = (1 << CHUNK_BITS) - 1;
   if ((v >> CHUNK_BITS) == (w >> CHUNK_BITS)) {
     return false;
+  } else if ((v & chunkMask) == (w & chunkMask)) {
```

Comment:
Again, this doesn't look like a valid inter-chunk dependency -- why can this return true?